Sketchfab-Exporter.pyp

```diff
@@ -688,7 +747,7 @@ This program comes with ABSOLUTELY NO WARRANTY. For details, please visit http:/
                 data['description'] = description
 
             if len(tags) != 0:
-                data['tags'] = tags
+                data['tags'] = 'cinema4d' + tags
```

Comment:
you should add a space probably otherwise we'd have `cinema4dmytags` instead of `cinema4d mytags`